src/Symfony/Component/Console/Command/LockableTrait.php

```diff
@@ -36,7 +36,7 @@
     private function lock($name = null, $blocking = false)
     {
         if (!class_exists(SemaphoreStore::class)) {
-            throw new RuntimeException('To enable the locking feature you must install the symfony/lock component.');
+            throw new LogicException('To enable the locking feature you must install the symfony/lock component.');
         }
 
         if (null !== $this->lock) {
```

Comment:
the `LogicException` below looks odd :)

Reply:
you mean we should return true instead? @jderusse WDYT?
@ro0NL up for a PR on 3.4 to trigger the discussion?

Reply:
on 2.8 actually? and unrelated to the Lock component, I was too fast :)

Reply:
just curious if it should be Runtime.. though Logic might be valid if the developer should call in specific order.

Reply:
yep, that makes sense also, asking devs to take care and pointing their mistake otherwise with LogicException

Reply:
This method should not be called twice. IMHO, this is a developer's misconception and should be a LogicException